tensorflow/lite/core/interpreter.h

```diff
@@ -115,7 +117,7 @@ class Interpreter {
   // used. Ownership of 'error_reporter' remains with the caller.
   // WARNING: Use of this constructor outside of an InterpreterBuilder is not
   // recommended.
-Interpreter(const std::wstring& flex_dll_path_, ErrorReporter* error_reporter = DefaultErrorReporter());
+Interpreter(const HMODULE& flex_dll, ErrorReporter* error_reporter = DefaultErrorReporter());
```

Comment:
Tiny nit. HMODULE is defined as a pointer. So no space is saved passing it by const reference vs passing it by value.